config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
It's possible, but they dont show up in the UI :(

Thats why I think we should a) do a corresponding doc update. I guess the docs request in the BZ is enough. Though it would be nice to involve the docs team right here on the PR.
@adahms ?